filebeat/recipes/default.rb

```diff
@@ -0,0 +1,23 @@
+template '/etc/yum.repos.d/filebeat.repo' do
+  source 'filebeat.repo.erb'
+  owner 'root'
+  group 'root'
+  mode '0755'
+end
```

Comment:
It looks to be provided by the [version of `yum` in the repo](https://github.com/chef-cookbooks/yum/tree/4e53fe10d713fe04219c81bb4a743eccba34c6a5).